devicetypes/tonesto7/nest-thermostat.src/nest-thermostat.groovy

```diff
@@ -441,20 +455,18 @@ def humidityEvent(humidity) {
 	def hum = device.currentState("humidity")?.value
 	if(!hum.equals(humidity)) {
         log.debug("UPDATED | Humidity is (${humidity}) | Original State: (${hum})")
-		sendEvent(name:'humidity', value: humidity, unit: "%", descriptionText: "Humidity is ${humidity}" , displayed: true, isStateChange: true)
+		sendEvent(name:'humidity', value: humidity, unit: "%", descriptionText: "Humidity is ${humidity}" , displayed: false, isStateChange: true)
     } else { Logger("Humidity is (${humidity}) | Original State: (${hum})") }
 }
 
 def presenceEvent(presence) {
-	def val = getPresence()
```

Comment:
Purely out of curiosity was their an issue with calling the getPresence() vs. the device.currentState?